src/components/contributors.tsx

```diff
@@ -31,6 +31,10 @@ const contributors: GitHubProfileProps[] = [
         userName: 'BernardAddo',
         role: 'Back-End Engineer',
     },
+    {
+        userName: 'Astraking',
+        role: 'Learner',
+    },
```

Comment:
I do not believe the role assigned to you in the document was learner. Please check and fix.